run-dask-process.sh

```diff
@@ -26,7 +26,7 @@ ARGS=$*
 function hasArg {
     (( ${NUMARGS} != 0 )) && (echo " ${ARGS} " | grep -q " $1 ")
 }
-VALIDARGS="-h --help scheduler workers"
+VALIDARGS="-h --help scheduler workers tcp ucx ucxib ucx-ib"
```

Comment:
Was looking at a better way to implement this. Something like a `--mode/cluster-type tcp` but went down a weird rabbit hole of `getopt` vs `getopts`. Open to suggestions/recommendations here

Reply:
yeah, I think the move to start using `getopt`/`getopts` is long overdue.  I'm in favor of either, but I'd need more time to look into each to see if there's pros and cons that apply.